### DISCUSSION TITLE
Is it possible to globally disable all bold output?

### DISCUSSION THREAD

**Discussion by igorsol** [starship/starship#2488] [Q&A]
By default many parts of starship's outputs are bold. I would prefer to not use bold at all. I understand how to configure text style per module but since there are so many modules it will be boring.
So is it possible to globally change default to not output bold text?

**Comment by davidkna** [comment]
No, but once #2309 happens, you should be able to easily accomplish this with search & replace.

**Reply by kryzar** [reply]
Would be nice to have a set of "global options" tho, like `no_bold = true`.
One other thing that could be nice is the ability to change style for all keywords such as 'via': I want to set them all to `dimmed white` and I would much prefer a oneliner-global option than having a gigantic config file like in #2309.

**Comment by vladimyr** [comment]
I've edited your question not to confuse anyone because disabling bold output is certainly possible (and boring 🙃 ) while disabling it globally is not.
